.codeclimate.yml

```diff
@@ -13,7 +13,7 @@ checks:
     config:
       threshold: 250
   method-complexity:
-    enabled: true
+    enabled: false
```

Comment:
hahaha I feel you mac, one day we'll get on top of these :D

Reply:
Actually I think I might keep this off for a long time 😝 . Got inspired by https://github.com/ipfs/go-ipfs/issues/4513#issue-283729397

Reply:
ahha agreed, we already have those in Rubocop as well